src/slack_message_formatter.ts

```diff
@@ -34,7 +34,7 @@ export function formatMessageHistory(
       );
     } else {
       return (
-        ':adult: *Helpline*  ' +
+        ':adult::skin-tone-4: *Helpline*  ' +
```

Comment:
Thoughts on using the emoji default yellow?

Reply:
sure!